Runtime/Localization/LocalizationConfig.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using UnityEngine;
+
+namespace Innoactive.Creator.Core.Internationalization
+{
+    /// <summary>
+    /// Localization configuration data structure. Will load all sources in an ordered way overwriting all
+    /// keys already existing.
+    /// </summary>
+    [CreateAssetMenu(fileName = "LocalizationConfig", menuName = "Innoactive/Localization Config", order = 0)]
+    public class LocalizationConfig : ScriptableObject
```

Comment:
This could be a SettingsObject<T> object

Reply:
no it can't be SettingsObject because there can be multiple.